src/test/php/PHPMD/RuleSetTest.php

```diff
@@ -91,7 +91,8 @@ private function createRuleSetFixture()
     {
         $ruleSet = new RuleSet();
 
-        for ($i = 0; $i < func_num_args(); ++$i) {
+        $funcNumArgs = func_num_args();
+        for ($i = 0; $i < $funcNumArgs; ++$i) {
             $ruleSet->addRule(new RuleStub(func_get_arg($i)));
```

Comment:
It would be way more clear this way:
```suggestion
        foreach (func_get_args() as $name) {
            $ruleSet->addRule(new RuleStub($name));
```
(And probably faster and less memory consuming)